doc/release-notes/release-notes-4.0.0-rc1.md

```diff
@@ -0,0 +1,280 @@
+      consensus: Remove canopyActive gate around GetActiveFundingStreamElements
+      consensus: Combine heartwoodActive conditionals
+      consensus: Add a placeholder for !canopyActive
+      consensus: Move overwinterActive rules ahead of saplingActive rules
+      consensus: Combine saplingActive conditionals
+      consensus: Move Sapling-disabled Overwinter rules above Sapling rules
+      consensus: Reorder Overwinter+!Sapling rules
+      consensus: Remove redundant contextual consensus rules
+      Add comment in lieu of redundant overwinter version check & fix tests.
+
+Larry Ruane (2):
+      flush wallet db (SetBestChain()) on clean shutdown
+      wallet: lock cs_main while accessing chainActive
+
+LongShao007 (1):
+      fix bug of bdb.mk
+
+Per Grön (11):
+      Get rid of implicit hidden dependencies between test .cpp files
+      Add missing #includes to test_block.cpp
+      Add actual header file for utilities in gtest/utils.cpp
+      Fix linkage issue with consts in primitives/block.h
+      Remove Checkpoints_tests.cpp
+      libsnark: Don't (implicitly) rely on other tests initializing the public params
+      Add missing libsnark initialization call
+      Don't clobber cwd in rpc_wallet_tests.cpp
+      Include header files within the source tree using "" instead of <>
+      Be consistent about what path to include bitcoin-config.h with
+      Be consistent with how to #include test data headers
+
+Pieter Wuille (1):
+      Add ctaes-based constant time AES implementation
+
+Sean Bowe (4):
+      Postpone boost 1.74.0 update
+      Postpone rust updates
+      make-release.py: Versioning changes for 4.0.0-rc1.
+      make-release.py: Updated manpages for 4.0.0-rc1.
+
+Taylor Hornby (5):
+      Implement system for postponing dependency updates.
+      Change release instructions to block the release when dependencies are not updated and not postponed.
+      Enforce pre-release dependency update check in make-release.py
+      Extend deadline for postponing dependency updates
+      Add new dependencies to updatecheck.py, add a flag we can use to have our CI test it.
+
+Ariel Gabizon (1):
+      explain expiry error
+
+bambam (2):
+      Added support for afl-clang-fast.
+      Added libfuzzer support.
+
+elbandi (3):
+      Allow configure params directory
+      Add paramsdir option for manpage
+      Throw error if -paramsdir not a valid directory
+
+fanquake (8):
+      depends: set OSX_MIN_VERSION to 10.10
+      doc: mention that macOS 10.10 is now required
+      scripted-diff: prefer MAC_OSX over __APPLE__
+      build: set minimum supported macOS to 10.12
+      depends: clang 6.0.1
+      depends: native_cctools 921, ld64 409.12, libtapi 1000.10.8
+      build: use macOS 10.14 SDK
+      doc: explain why passing -mlinker-version is required
+
+noname45688@gmail.com (2):
+      Updating to Python 3
+      Update to Python 3
+
+Jack Grigg (2):
```

Comment:
Not sure why this wasn't merged with the other "Jack Grigg" section above.

Reply:
Did it manually for now.